runtime/bcverify/bcverify.c

```diff
@@ -867,16 +867,16 @@ mergeStacks (J9BytecodeVerificationData * verifyData, UDATA target)
 	UDATA maxIndex = J9_ARG_COUNT_FROM_ROM_METHOD(romMethod) + J9_TEMP_COUNT_FROM_ROM_METHOD(romMethod);
 	U_32 *bytecodeMap = verifyData->bytecodeMap;
 	UDATA i = 0;
-	UDATA stackIndex;
+	UDATA stackIndex = bytecodeMap[target] >> BRANCH_INDEX_SHIFT;
 	IDATA rewalk = FALSE;
 	IDATA rc = BCV_SUCCESS;
-	UDATA *targetStackPtr, *targetStackTop, *sourceStackPtr, *sourceStackTop, *sourceStackTemps;
+	UDATA *targetStackPtr = NULL;
+	UDATA *targetStackTop = NULL;
+	UDATA *sourceStackPtr = NULL;
+	UDATA *sourceStackTop = NULL;
+	UDATA *sourceStackTemps = NULL;
 	J9BranchTargetStack *liveStack = (J9BranchTargetStack *) verifyData->liveStack;
-	J9BranchTargetStack *targetStack;
-
-
-	stackIndex = bytecodeMap[target] >> BRANCH_INDEX_SHIFT;
-	targetStack = BCV_INDEX_STACK (stackIndex);
+	J9BranchTargetStack *targetStack = BCV_INDEX_STACK (stackIndex);
```

Comment:
Actually, while we're here, let's remove the extra blank. Do it after testing passes and I'll merge.

Reply:
which blank line do you want removed?  I only see one between the end of the variable declarations and the first tracepoint.

Reply:
The one between the macro and the bracketted arg.

Reply:
Click.  I read that as "blank line".  Fixed now